nextflow_aws_batch_trigger.py

```diff
@@ -19,23 +19,26 @@ def create_launch_template(
     launch_template_names = [i["LaunchTemplateName"] for i in launch_templates]
 
     if launch_template_name not in launch_template_names:
-        user_data = f"""\
-                    MIME-Version: 1.0
-                    Content-Type: multipart/mixed; boundary= ==MYBOUNDARY==
-
-                    --==MYBOUNDARY==
-                    Content-Type: text/x-shellscript; charset=us-ascii
-
-                    #!/bin/bash
-                    #!/bin/bash -xe
-                    sudo amazon-linux-extras install epel -y
-                    sudo yum install s3fs-fuse -y
-                    mkdir {s3fs_mount}
-                    mkdir {result_location}
-                    chmod 777 {s3fs_mount}
-                    sudo s3fs {s3_bucket} {s3fs_mount} -o allow_other -o umask=000 -o iam_role=auto
-
-                    --==MYBOUNDARY==--"""
+        user_data = (
+            """MIME-Version: 1.0\nContent-Type: multipart/mixed; boundary="==MYBOUNDARY=="\n\n--==MYBOUNDARY==\n"""
+            """Content-Type: text/cloud-config; charset="us-ascii"\n\npackages:\n- jq\n- aws-cli\n\n"""
+            """runcmd:\n"""
+            """- amazon-linux-extras install epel -y\n"""
+            """- yum install s3fs-fuse -y\n"""
+            """- /usr/bin/aws configure set region $(curl http://169.254.169.254/latest/meta-data/placement/region)\n"""
```

Comment:
Instead of calling the url listed, use the `region_name` we pass in.

Reply:
have passed variable 'region_name' now on line 28.